### PR TITLE
Logo/favicon for dementia study

### PR DESCRIPTION
**Pull request by achen2401**
address https://www.pivotaltracker.com/story/show/183063815

Changes include:
- add new environment variable, i.e. `VUE_APP_PROJECT_ID`, to identify a specific project
- new logo and favicon for the dementia study
- update code to allow project specific logo and theme

### CODE REVIEW

File: src/plugins/vuetify.js [hunk 1]
```diff
@@ -2,7 +2,14 @@ import Vue from 'vue'
 import Vuetify from 'vuetify'
 import 'vuetify/dist/vuetify.min.css';
 import themes from '../context/themes';
+import {getEnv} from '../util/util';
 
+const getAppTheme = () => {
+    const projectID = getEnv("VUE_APP_PROJECT_ID");
+    if (themes[projectID] && themes[projectID].app) return themes[projectID]
+                                                      .app;
```
Comment (ivan-c) [inline src/plugins/vuetify.js:10]: Accidental newline?
```suggestion
    if (themes[projectID] && themes[projectID].app) return themes[projectID].app;
```

**Review by ivan-c — APPROVED** [review]
Looks good! Think I found a typo, but ready to merge either way

File: README.md [hunk 2]
```diff
@@ -84,6 +84,7 @@ Parameters for the app are stored in [environment variables](http://man7.org/lin
 | `VUE_APP_FHIR_RESOURCES` | For specifying which FHIR resource(s) to load for the selected patient. | `Condition,Procedure,Observation,QuestionnaireResponse` |
 | `VUE_APP_AUTH_SCOPES` | For allowing the app to specify the delegation of a specific set of access rights via launch context. see [App Launch: Scopes and Launch Context](https://build.fhir.org/ig/HL7/smart-app-launch/scopes-and-launch-context.html) | `profile roles email patient/*.read openid fhirUser patient/QuestionnaireResponse.write` |
 | `VUE_APP_CONF_API_URL` | URL from which the app can retrieve launch context, i.e. launching as a confidential client | ''
+| `VUE_APP_PROJECT_ID` | Project ID that uniquely identifies the project.  The app will display project specific logo and theme based on the ID | 'DEM'
```
Comment (mcjustin) [inline README.md:87]: @achen2401  is it too late to change all "DEM" references to "DCW"?

Comment (achen2401) [reply README.md]: @mcjustin I updated all "DEM" references to "DCW".  Just want to note that, do we need to make the same change for the [environments repo](https://github.com/uwcirg/dem-environments/tree/main/dev) as well?

Comment (mcjustin) [reply README.md]: Thanks @achen2401 for doing that, and for flagging environments. I'll create a PR over there FYI @ivan-c , might as well nip it in the bud

**Review by pbugni — APPROVED** [review]
Looks great @achen2401  - sorry for the review delay!

**Pull request merged**